versions.lock

```diff
@@ -1,31 +1,39 @@
 # Run ./gradlew --write-locks to regenerate this file
-ant:ant:1.6.5 (1 constraints: bb0a41d8)
-antlr:antlr:2.7.7 (2 constraints: 36167e02)
+ant:ant:1.6.5 (2 constraints: 2d1539e1)
 aopalliance:aopalliance:1.0 (1 constraints: 170a83ac)
-asm:asm:3.1 (2 constraints: 4f19c3c6)
+asm:asm:3.1 (3 constraints: 251fd7ad)
+asm:asm-commons:3.1 (1 constraints: 9c0f1f7a)
+asm:asm-tree:3.1 (1 constraints: 2307035c)
+ch.qos.logback:logback-classic:1.0.9 (3 constraints: d3258a88)
+ch.qos.logback:logback-core:1.0.9 (4 constraints: dd32435a)
+co.cask.tephra:tephra-api:0.6.0 (3 constraints: 0828ded1)
+co.cask.tephra:tephra-core:0.6.0 (2 constraints: 831cd90d)
+co.cask.tephra:tephra-hbase-compat-1.0:0.6.0 (1 constraints: 370d6920)
```

Comment:
Do we need this HBase dependency? What is tephra used for?

Reply:
tephra is a "globally consistent transactions on top of Apache HBase.", introduced by Hive Metastore, I don't think it is needed, will remove it.